cps/src/main/js/workflow-editor.js

```diff
@@ -48,7 +48,34 @@ jenkinsJSModules.import('ace-editor:ace-editor-122')
                 });
 
                 editor.on('blur', function() {
-                    textarea.trigger('change');
```

Comment:
When `sandbox` is _unchecked_ then any change to `script` requires `ScriptApproval` to be consulted for `FormValidation`, in case a non-admin is editing the script.

I do not get why you cannot simply restore the call to `trigger` here, so `doCheckScript` will verify script approval exactly as it did before this PR, while `doCheckScriptJson` will handle line-by-line Groovy errors.